lms/envs/common.py

```diff
@@ -2174,6 +2174,12 @@
     'DEFAULT_PAGINATION_CLASS': 'openedx.core.lib.api.paginators.DefaultPagination',
     'PAGE_SIZE': 10,
     'URL_FORMAT_OVERRIDE': None,
+    'DEFAULT_THROTTLE_CLASSES': (
+        'rest_framework.throttling.UserRateThrottle',
+    ),
+    'DEFAULT_THROTTLE_RATES': {
+        'user': '50/minute',
```

Comment:
Let's start with 60/minute (1/second) -- we can always adjust this value if the constraint is too tight.